.fixtures.yml

```diff
@@ -20,6 +20,8 @@ fixtures:
     pe_r10k: "git@github.com:puppetlabs/puppetlabs-pe_r10k.git"
     pe_staging: "git@github.com:puppetlabs/puppetlabs-pe_staging.git"
     puppet_enterprise: "git@github.com:puppetlabs/puppetlabs-puppet_enterprise.git"
+    stunnel: "git@github.com:puppetlabs/puppetlabs-stunnel.git"
```

Comment:
That's sort of the point. The module from the Forge is what will be used in the classroom.